Gopistol/demo/src/main/java/com/example/demo/comment/controller/commentRequest/CommentRequestDTO.java

```diff
@@ -0,0 +1,11 @@
+package com.example.demo.comment.controller.commentRequest;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class CommentRequestDTO {
+
+  String comment;
+}
```

Comment:
```suggestion
public record CommentRequestDTO(String comment) {}
```
record로 처리할 수 있으면 `record`로 하고 일반 클래스라면 `final` 넣어주기.
`jacksonConverter`가 문제라면 `@JsonCreator` 같은 어노테이션 확인해보기